test/metabase/models/card_test.clj

```diff
@@ -291,3 +291,40 @@
                clojure.lang.ExceptionInfo
                #"Invalid Field Filter: Field \d+ \"VENUES\"\.\"NAME\" belongs to Database \d+ \"test-data\", but the query is against Database \d+ \"sample-dataset\""
                (db/update! Card card-id bad-card-data))))))))
+
+(deftest action-creation-test
+  (testing "actions are created when is_write is set"
+    (testing "during create"
+      (let [{card-id :id} (db/insert! Card (assoc (tt/with-temp-defaults Card) :is_write true))
+            {:keys [action_id card_id] :as qa-rows} (db/select-one QueryAction :card_id card-id)]
```

Comment:
You should use `with-temp` here which will delete the Cards after the test. Otherwise if you run these tests from the REPL you will end up with a ton of new garbage Cards in your application DB each time you run the test. Generally the tests are supposed to clean up after themselves. Another option is `with-model-cleanup` if you still want to be able to use `insert!` manually instead of having `with-temp` do it for you. `with-temp` ultimately calls `insert!` tho so you can use it for this stuff